csrc/emulator.cc

```diff
@@ -82,12 +83,14 @@ int main(int argc, char** argv)
 
   // reset for several cycles to handle pipelined reset
   for (int i = 0; i < 10; i++) {
+    done_reset = false;
     tile->reset = 1;
     tile->clk = 0;
     tile->eval();
     tile->clk = 1;
     tile->eval();
     tile->reset = 0;
+    done_reset = true;
   }
```

Comment:
you should be able to remove the done_reset = false, as the global will be so initialized, and then you can move done_reset = true to the outside of the loop, which makes the intent clearer.  looks good otherwise!